todo-mvc-kitchensink/src/TodoAppContext.ts

```diff
@@ -0,0 +1,168 @@
+import { Evented } from '@dojo/core/Evented';
```

Comment:
This is an encapsulated Todo state class that is passed as a context that gets injected into `Containers` (could rename to `TodoAppState` if it makes more sense)